src/pages/registration/register.page.ts

```diff
@@ -202,10 +198,28 @@ export class RegisterPage implements OnInit { // this part of registration is fo
                     .subscribe(
                       results => {
                         loading.dismiss().then(() => {
-                          this.milestone_id = data[0].id;
-                          self.cacheService.setLocal('milestone_id', data[0].id);
-                          self.navCtrl.push(TabsPage).then(() => {
-                            window.history.replaceState({}, '', window.location.origin);
+                          // results[0] game API data
+                          this.gameID = results[0].Games[0].id;
```

Comment:
@DamengRandom take note, same code repeated in this file too. same for: [*login.ts*, *magic-link.ts*]

Reply:
Yup sure, I will refactor it, thx @trtshen ^_^